ScrollingTabControllerExample/ScrollingTabControllerExample/ViewController.swift

```diff
@@ -35,6 +35,26 @@ class ViewController: ScrollingTabController {
         self.buildViewControllers()
         self.tabSizing = .sizeToContent
         self.tabView.centerSelectTabs = true
+
+        let nextBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FastForward, target: self, action: #selector(jumpAhead))
+        let previousBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Rewind, target: self, action: #selector(jumpBack))
+        navigationItem.rightBarButtonItems = [nextBarButtonItem, previousBarButtonItem]
+    }
+
+    func jumpAhead() {
```

Comment:
do you want to capture the sender? I guess you don't need it but it's nice how including it implies that it's tied to a button action!